docs/docs/en/guide/task/seatunnel.md

```diff
@@ -26,17 +26,16 @@ Click [here](https://seatunnel.apache.org/) for more information about `Apache S
 - SEATUNNEL_ENGINE
 - Deployment mode: specify the deployment mode, `cluster` `local`
 
-          > Click [here](https://seatunnel.apache.org/docs/2.3.3/command/usage) for more information on the usage of 
-
-`Apache SeaTunnel command`
+  > Click [here](https://seatunnel.apache.org/docs/2.3.3/command/usage) for more information on the usage of Apache SeaTunnel command`
```

Comment:
Hi @liunaijie , how about update the doc version to 2.3.8?

Reply:
We can delete the version path, update to 
https://seatunnel.apache.org/docs/command/usage/

This link will always avaliable.

Reply:
Yes, this way would be better.

Reply:
I will update it